ts/sagas/wallet.ts

```diff
@@ -808,6 +830,15 @@ function* completionHandler(
 ) {
   // -> it should proceed with the required operations
   // and terminate with the "new payment" screen
+
+  // Retrieve the configured PIN from the keychain
+  const storedPin: SagaCallReturnType<typeof getPin> = yield call(getPin);
+  if (storedPin.isSome()) {
```

Comment:
this is awkward since you cannot reach this point unless the PIN is set, the PIN should be passed to the saga as parameter